### DISCUSSION TITLE
Why my contributions to a community aren't showing on my profile?

### DISCUSSION THREAD

**Discussion by gisellevazquez** [community/community#113281] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I had contributed for months to a community with documentation via gitbook, and my account are connected, I appear on this list: https://github.com/CDPR-Modding-Documentation/Cyberpunk-Modding-Docs/graphs/contributors but the contributions I do, aren't showing on my profile:
![image](https://github.com/community/community/assets/84458091/8eb1a775-4e78-41cb-9afa-a3f1e6d3ae3e)

I did a contribution yesterday and should appear there, but nothing, what is wrong? is any configuration missing?

**Comment by AidenJamesYt** [comment]
this? 
![image](https://github.com/community/community/assets/60198701/dfcc2116-af2c-41f2-93e0-d6f409a56a75)

**Comment by gisellevazquez** [comment]
Oh they are appearing now!! It probably was because I had my email like "private" that, if someone needs that on the future [](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Comment by gisellevazquez** [comment]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile
